watertap/unit_models/zero_order/tests/test_anion_exchange_zo.py

```diff
@@ -0,0 +1,247 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+Tests for zero-order anion exchange model
+"""
+import pytest
+from io import StringIO
+
+from pyomo.environ import (
+    check_optimal_termination, ConcreteModel, Constraint, value, Var, Param)
+from pyomo.util.check_units import assert_units_consistent
+
+from idaes.core import FlowsheetBlock
+from idaes.core.util import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.testing import initialization_tester
+
+from watertap.unit_models.zero_order import AnionExchangeZO
+from watertap.core.wt_database import Database
+from watertap.core.zero_order_properties import WaterParameterBlock
+
+solver = get_solver()
+
+
+class TestAnionExchangeZO_w_o_default_removal:
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.db = Database()
+
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+        m.fs.params = WaterParameterBlock(
+            default={"solute_list": ["tds"]})
+
+        m.fs.unit = AnionExchangeZO(default={
+            "property_package": m.fs.params,
+            "database": m.db})
+
+        m.fs.unit.inlet.flow_mass_comp[0, "H2O"].fix(10000)
+        m.fs.unit.inlet.flow_mass_comp[0, "tds"].fix(1)
+
+        return m
+
+    @pytest.mark.unit
+    def test_build(self, model):
+        assert model.fs.unit.config.database == model.db
```

Comment:
We should probably be using `is` for this check. I see it as equals elsewhere, but I think we mean that `model.fs.unit.config.data` is the object `model.db`, and not equivalent to `model.db`.

Since nobody has defined `__eq__` on `Database` this is the same, for now.